Add initial 10 year post

ericholscher opened the pull request:
This is just a first draft, but I think the overall structure has come together. 

<!-- readthedocs-preview readthedocs-about start -->
----
📚 Documentation preview 📚: https://readthedocs-about--324.org.readthedocs.build/

<!-- readthedocs-preview readthedocs-about end -->

agjohnson commented on content/posts/10-year-anniversary.md:
```diff
@@ -0,0 +1,65 @@
+
+A few of the most important lessons from the last 10 years:
+
+* You don't get extra points for being open source or bootstrapped. People will compare you to proprietary, venture-backed services that have raised many millions of dollars. **We compete by focusing on documentation, a niche too small for most venture-backed businesses**. Venture-backed companies have to keep adding unrelated features in order to grow into other areas, and we can focus on our co ...
```
I'd say we actually do get _a fair amount_ of points for being open source at least. There is a lot of good will there because we are open source and support open source, and customers quite often come over from community. But this still makes the most sense framed around just bootstrapping too.

 ```suggestion
* You don't get extra points for being bootstrapped. People will compare you to proprietary, venture-backed services that have raised many millions of dollars. **We compete by focusing on documentation, a niche too small for most venture-backed businesses**. Venture-backed companies have to keep adding unrelated features in order to grow into other areas, and we can focus on our core competency.
```

There is something to say about open source though.

agjohnson commented on content/posts/10-year-anniversary.md:
```diff
@@ -0,0 +1,65 @@
+A few of the most important lessons from the last 10 years:
+
+* You don't get extra points for being open source or bootstrapped. People will compare you to proprietary, venture-backed services that have raised many millions of dollars. **We compete by focusing on documentation, a niche too small for most venture-backed businesses**. Venture-backed companies have to keep adding unrelated features in order to grow into other areas, and we can focus on our co ...
+* Keeping trust in the community is the most important thing, because we're nothing without our users. It's a cliche, but trust is hard to earn and easy to lose. We continually think about how to balance sustainability with our the values of our users, and have made many decisions over the years about having "enough". **We can't maximize profit and continue to keep the trust of our community.**
```
```suggestion
* Keeping trust in the community is the most important thing, because we're nothing without our users. It's a cliche, but trust is hard to earn and easy to lose. We continually think about how to balance sustainability with the values of our users, and have made many decisions over the years about having "enough". **We can't maximize profit and continue to keep the trust of our community.**
```

ericholscher commented on content/posts/10-year-anniversary.md:
```diff
@@ -0,0 +1,65 @@
+## Next steps
+
+Read the Docs started only supporting Sphinx, and now supports any documentation tool that generates HTML output.
+We've also finished up a large "magic removal" branch that has removed our modification of the documentation build process,
+reducing the confusion that users hit building locally versus on Read the Docs.
+On top of that,
+we're layering our [Addons system](https://docs.readthedocs.io/en/stable/addons.html),
+which providers features for documentation readers on top of any tool.
```
I think that's what this is saying?

agjohnson replied:
I meant this could discuss our vision a bit more broadly. Right now it reads more like we're describing a recent sprint -- ie the magic removal branch is nice but doesn't really feel like a vision for RTD.

ericholscher replied:
I mean, changing the build system to not modify user configs is a huge change, and is definitely part of the new vision for the product where we aren't changing user HTML at all.

ericholscher commented:
Alright, I've addressed most of the feedback -- I'd love to get this published today, or feel free to merge it while I'm gone and ship it out on the socials 🎉

ericholscher commented on content/posts/10-year-anniversary.md:
```diff
@@ -0,0 +1,65 @@
+## Our sustainability model
+
+We tried a number of different ways to make a sustainable open source project, really a service, over the years.
+The model that we've found that has worked for us currently is a version of the classic open source model used by GitHub, Travis CI, and others:
```
I removed the examples, since I feel like nobody else is really doing quite the same thing -- I do think Travis did it the most with the 2 sites running on OSS, but not sure of other examples (not sure that's a good thing 😆)

agjohnson replied:
Yeah, we definitely were following Travis' lead for sure. It only feels sad comparing because of their PE buyout and all of the fallout there.

humitos commented on content/posts/10-year-anniversary.md:
```diff
@@ -0,0 +1,71 @@
+This leaves us with a much simpler system:
+
+* Our build system [now supports any documentation tool](https://docs.readthedocs.io/en/stable/build-customization.html) and should work the same as other build environments.
+* For built documentation we've added our [Addons system](https://docs.readthedocs.io/en/stable/addons.html) which providers features for documentation readers on top of any tool.
```
```suggestion
* For built documentation we've added our [Addons system](https://docs.readthedocs.io/en/stable/addons.html) which provides features for documentation readers on top of any tool.
```

humitos commented on content/posts/10-year-anniversary.md:
```diff
@@ -0,0 +1,71 @@
+We tried a number of different ways to make a sustainable open source project, really a service, over the years.
+The model that works for us is a take on the classic "open core" model, but applied to a service versus the code base:
+
+* We offer a free service (Read the Docs Community) for open source projects, supported by a single non-tracking ad on the documentation.
```
```suggestion
* We offer a free service (Read the Docs Community) for open source projects, supported by [a single non-tracking ad](https://docs.readthedocs.io/en/stable/advertising/) on the documentation.
```

humitos reviewed (approved):
Looks ready to be publish in my opinion.

humitos commented:
> I'd love to get this published today, or feel free to merge it while I'm gone and ship it out on the socials 🎉

@agjohnson this post is ready to go to me. Should I merge it?

agjohnson commented on content/posts/10-year-anniversary.md:
```diff
@@ -0,0 +1,71 @@
+title: 10 years of attempting to make open source sustainable
+date: 2024-08-15
```
```suggestion
date: 2024-08-26
```

agjohnson commented:
Let's merge it monday, I've bumped the date up to prepare.

humitos commented:
Merging...
